homeassistant/components/media_player/russound_rnet.py

```diff
@@ -79,10 +78,34 @@ def __init__(self, hass, russ, sources, zone_id, extra):
         """Initialise the Russound RNET device."""
         self._name = extra['name']
         self._russ = russ
-        self._state = STATE_OFF
         self._sources = sources
         self._zone_id = zone_id
-        self._volume = 0
+
+        self._state = None
+        self._volume = None
+        self._source = None
+
+        self.update()
```

Comment:
Instead of calling `self.update()` at the end of `__init__`, add `True` to the end of `add_devices` here: https://github.com/home-assistant/home-assistant/pull/5756/files#diff-94a55579c81fd6ed2ecfc6300f04aca5R68

Example: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/switch/dlink.py#L56

There's still a lot of devices that are doing this the old way (including nad that I linked), but this is the preferred new way to do the initial update.